### PR TITLE
fix: remove padding from borderless stat-tabs, extend varTheme

### PR DESCRIPTION
**Pull request by jhoward-eightfold**
## SUMMARY:
When turning off border for StatTabs, remove the padding as designers will most often want the tabs flush with the surrounding content.

Additional enhancements to varTheme support.

## CHANGE TYPE:

- [x] Bugfix Pull Request
- [ ] Feature Pull Request

## TEST COVERAGE:

- [ ] Tests for this change already exist
- [ ] I have added unittests for this change

## TEST PLAN:
Visual inspection of StatTabs to ensure they render as desired.

### CODE REVIEW

File: src/styles/themes/_definitions.scss [hunk 1]
```diff
@@ -54,15 +54,15 @@ $color-tertiary-blue: var(--blue-color-60);
 $color-tertiary-violet: var(--violet-color-60);
 $color-tertiary-grey: var(--grey-color-60);
 
-$text-color-primary: var(--grey-color-90);
-$text-color-secondary: var(--grey-color-80);
+$text-color-primary: var(--text-primary-color);
+$text-color-secondary: var(--text-secondary-color);
```
Comment (jhoward-eightfold) [inline src/styles/themes/_definitions.scss:58]: previous value of $text-color-secondary did not match that of --text-secondary-color... do we think this was intentional and I should undo this change or should we consider this a fix?

Comment (jhoward-eightfold) [reply src/styles/themes/_definitions.scss:58]: Probably not a huge deal, only one implementation point in table.module.scss:
```
    &-filter-trigger {
     ...

      &:hover {
        color: $text-color-secondary;
        background: $table-header-filter-active-background-color;
      }
```

File: src/styles/themes/_default-theme.scss [hunk 2]
```diff
@@ -208,6 +214,25 @@
   --tab-underline: var(--border-color);
   // ------ Tabs theme ------
 
+  // ------ Stat tab theme ------
```
Comment (dkilgore-eightfold) [inline src/styles/themes/_default-theme.scss:217]: Nice.

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: src/components/Tabs/tabs.module.scss [hunk 3]
```diff
@@ -228,21 +229,38 @@
       border-color: var(--border);
     }
 
+    &:not(.bordered) {
+      padding: 0;
+
+      &.divider {
+        padding: 0;
+
+        .tab {
```
[the remark this answers is not in the document]
Comment (jhoward-eightfold) [reply src/components/Tabs/tabs.module.scss]: Done

**Review by dkilgore-eightfold — APPROVED** [review]

**Review by dkilgore-eightfold — APPROVED** [review]
